src/services/billing-services/list-payment-service.js

```diff
@@ -0,0 +1,67 @@
+import { AxiosHttpClientAdapter, parseHttpResponse } from '../axios/AxiosHttpClientAdapter'
+import { makePaymentBaseUrl } from './make-payment-base-url'
+
+export const listPaymentService = async () => {
+  let httpResponse = await AxiosHttpClientAdapter.request({
+    url: `${makePaymentBaseUrl()}/credit_cards?page_size=200`,
+    method: 'GET'
+  })
+
+  httpResponse = adapt(httpResponse)
+
+  return parseHttpResponse(httpResponse)
+}
+
+const formatDate = (month, year) => {
+  const formatMonth = month < 10 ? `0${month}` : month
+
+  return `${formatMonth}/${year}`
+}
+
+const isExpired = (month, year) => {
```

Comment:
Usually (by convention) the `is` prefix is used when we return a boolean. In this case we are returning a string. Maybe rename the method to
```suggestion
const getExpiredString = (month, year) => {
```
?

Reply:
okay